dungeon/src/contrib/components/HealthComponent.java

```diff
@@ -242,4 +249,41 @@ public void godMode(boolean status) {
   public boolean godMode() {
     return this.godMode;
   }
+
+  /**
+   * Resolve and cache the {@link Entity} that owns this {@link HealthComponent}.
+   *
+   * <p>This method scans the current level entities to find the one that contains this component.
+   * On first successful lookup, the result is cached to avoid repeated full scans on subsequent
+   * calls.
+   *
+   * @return An {@link Optional} containing the owning {@link Entity} if present; otherwise {@link
+   *     Optional#empty()}.
+   */
+  private Optional<Entity> owner() {
+    if (ownerCache.isPresent()) return ownerCache;
+    ownerCache =
+        Game.levelEntities()
+            .filter(e -> e.fetch(HealthComponent.class).map(hc -> hc == this).orElse(false))
+            .findFirst();
+    return ownerCache;
+  }
+
+  /**
+   * Default handler that is invoked when the associated entity receives damage.
+   *
+   * <p>Applies a short visual feedback (a blink effect) to the owning entity if it has a {@link
+   * core.components.DrawComponent}.
+   *
+   * @param cause The entity that caused the damage; may be {@code null}.
+   * @param damage The {@link Damage} instance describing the received hit.
+   */
+  private void defaultOnHit(Entity cause, Damage damage) {
+    owner()
+        .filter(e -> e.isPresent(core.components.DrawComponent.class))
+        .ifPresent(
+            e ->
+                SkillTools.blink(
+                    e, HIT_BLINK_COLOR_ARGB, HIT_BLINK_DURATION_MS, HIT_BLINK_ITERATIONS));
+  }
```

Comment:
mach das mal in `HealthSystem#applyDamage` 
wenn `dmgAmount>0` 
Dann ist das kein Callback mehr, sondern Framework Beahvior, wenn eine Entität schaden bekommt.